rules/know-the-basic-terms-for-a-video/rule.md

```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+title: Production -  Do you know the basic terms for a video?
+uri: know-the-basic-terms-for-a-video
+authors:
+  - title: Adam Cogan
+    url: https://www.ssw.com.au/people/adam-cogan
+  - title: Andrew Forsyth
+    url: https://ssw.com.au/people/andrew-forsyth
+  - title: Jonty Gardner
+    url: https://www.ssw.com.au/people/jonty-gardner
+  - title: Landon Maxwell
+    url: "https://www.ssw.com.au/people/landon-maxwell "
+  - title: Sofie Hong
+    url: https://www.ssw.com.au/people/sofie-hong
+related:
+  - use-cutaways
+created: 2022-10-31T12:40:29.280Z
+guid: 9a3bc559-98d6-4119-88b5-0b7f4f342f6b
+---
+Understanding basic video terminology allows for better communication in video creation. Using basic video terminology can make our video content more engaging. It's important and useful for anyone involved in video production to know the basic terminology.
+
+## Filming Terminology
```

Comment:
1. Change from "Terminology" to "Terminologies"

Reply:
Done